pkg/cmd/server/apis/config/helpers.go

```diff
@@ -238,6 +238,10 @@ func GetMasterFileReferences(config *MasterConfig) []*string {
 		refs = append(refs, &config.AuthConfig.RequestHeader.ClientCA)
 	}
 
+	for _, wta := range config.AuthConfig.WebhookTokenAuthenticators {
```

Comment:
Can you use the index instead?  I have trouble keeping my head straight on when copies are made and the originals are used.  The rest of this method uses `for k := range foo; append(&foo[k])`

Reply:
ok